javascript/js-loops_forof-list/js/index.js

```diff
@@ -18,5 +18,26 @@ const programmingLanguages = [
 ];
 
 // --v-- write/change code here --v--
-
+for (let element of programmingLanguages) {
+  let list = document.createElement("li");
+  ol.append(list);
+  list.innerHTML = element;
```

Comment:
I guess, this works, too. So just a tiny comment since the task asked for "What was the property to set the text of an HTML element again?": you could also use textContent instead of innerHTML. Besides that: well done!